src/main/java/Infrastructure/Connection.java

```diff
@@ -1,4 +1,4 @@
-package echoserver;
+package Infrastructure;
```

Comment:
Cool. I am excited to see where this goes.  I think it is conventional to use lowercase package names in Java. I could very well be wrong about that.

Reply:
Oh yes.  Will modify.